### DISCUSSION TITLE
🚀极简拧水龙头教程 - 无需交易记录或账户余额

### DISCUSSION THREAD

**Discussion by zer0fire** [openbuildxyz/Web3-Frontend-Bootcamp#897] [Materials]
首先要确认一下，这个方法还是需要跑电脑算力的，我目测了一下如果只是 1-5 个 worker 的话应该相当于打开一个 PhotoShop 的消耗，当然不算准确。

测试币水龙头网址
https://sepolia-faucet.pk910.de/

需要先去这个地址申请 GitCoin Passport，也就是吉特币护照，比较好过，几个社交媒体、Github 一通认证怎么也凑够 2 分就可以了 
<img src="https://github.com/openbuildxyz/Web3-Frontend-Bootcamp/assets/26370252/c119ff88-4c4d-461c-8dc8-2f68b5c3f801" style="width: 50px !important; height: 50px;" />

然后开挖，记得切换自己的钱包到 Sepolia 网络，并且在这里输入你的公钥（当然你输入我的我也不介意，哈哈哈，开玩笑）
![image](https://github.com/openbuildxyz/Web3-Frontend-Bootcamp/assets/26370252/a3db0d24-6b99-401d-b198-22de45776053)

切换 Sepolia，我这里用 MetaMask 做示例
![image](https://github.com/openbuildxyz/Web3-Frontend-Bootcamp/assets/26370252/f5ceafcb-cd31-4232-a0db-f084c1bf51a6)
![image](https://github.com/openbuildxyz/Web3-Frontend-Bootcamp/assets/26370252/55dc80d0-755e-4e38-8039-6f68accfc91f)

最后看这个地方，有个`worker`，可以选择算力的消耗大小，我这里选了一个慢慢挖
![image](https://github.com/openbuildxyz/Web3-Frontend-Bootcamp/assets/26370252/7b215e85-5901-43ea-bce6-3bbaa638d441)

注意这里的挖矿是有**最小限制**的，我第一次没达到最小限制直接给我搞失败了，白挖了😭
![image](https://github.com/openbuildxyz/Web3-Frontend-Bootcamp/assets/26370252/f8ed34e3-7bbe-4852-b5b4-596702cf0e57)

看到这个界面就成功啦
![image](https://github.com/openbuildxyz/Web3-Frontend-Bootcamp/assets/26370252/86a3ed75-9190-45b1-a286-73c4b6e6c0b9)

如果笔记对你有帮助的话，可以分我的测试账号一点点测试币🤏🏻，也可以测试一下转账之类的效果，万分感谢！😁
我的钱包地址：0xF69618C9c30AF9d8deAD34d3Db2570ed177308B7

**Comment by hu-qi** [comment]
Get！Good job！Thanks！
<img width="809" alt="image" src="https://github.com/openbuildxyz/Web3-Frontend-Bootcamp/assets/17986122/c14aa7da-320a-4097-8ea5-e656cb8adfee">
